src/dmd/file_manager.d

```diff
@@ -70,6 +70,11 @@ nothrow:
             return sc;
         scope(exit) FileName.free(sc.ptr);
 
+        const speculativeHeader = FileName.forceExt(filename, h_ext);
```

Comment:
what is this for?

Reply:
I actually can't remember (clearly I'm getting old). It's probably a bit of debug scaffolding that I forgot about that survived a rebase?

I'll get rid of it in the morning.